ush/config.sh.RTMA_CONUS_3km

```diff
@@ -95,6 +95,9 @@ l_PBL_pseudo_SurfobsT=.false.
 l_PBL_pseudo_SurfobsQ=.false
 i_use_2mQ4B=0
 i_use_2mT4B=0
+i_T_Q_adjust=2
+l_rtma3d=.true.
+i_precip_vertical_check=0
```

Comment:
Why is the value of "i_precip_vertical_check" different for CONUS vs. NA?

Reply:
This parameter crashes the cloud analysis. So I turned it off for CONUS. The one for NA_3km is just a placeholder and we did not test it yet